public/electron.js

```diff
@@ -225,3 +230,52 @@ app.on('activate', function () {
     createWindow()
   }
 })
+
+function addLedgerListeners(mainWindow) {
+  ipcMain.on("ledger:getVersion", () => {
```

Comment:
I refactored the code to use a `for` loop to add all methods since they're very similar. I didn't use a single method because there are specific calls and event names, but the code is simpler now. Let me know what you think.

Reply:
Great. Thanks!